### ISSUE TITLE
Non English Characters are not supported

### ISSUE THREAD

**Issue by deepjyoti30** [deepjyoti30/simber#3]
# Bug report

## Checklist

- [x] I have searched for similar bugs in the issues.
- [x] I will add the bug label to this issue.

## Version Info

- Simber version: `latest`

## Description

Non english characters are raising a `UnicodeEncodeError`.

**Comment by Cwavs** [comment]
This issue is pretty breaking for me, and I would imagine others, because the most common way to use the command line on windows is cmd imo, being unable to handle anything with non English characters is a pretty big issue on one of the most popular platforms. In the case of ytmdl, downloading anime music is hard as its common for titles to contain Japanese, and this error breaks it.

**Comment by deepjyoti30** [comment]
@Cwavs Thanks for letting me know about it. I will test this on `Windows` to see if I can reproduce this. My access to an Windows machine is scarce so it's hard for me to debug on `Windows` but I will try anyhow!

**Comment by Cwavs** [comment]
I'm having one of my friends attempt to recreate the issue as well so we can be sure its not just some esoteric issue on my end.

**Comment by deepjyoti30** [comment]
Awesome, let me know how that works out!

**Comment by xotakfr** [comment]
Hey, I'm also affected by this problem and I wrote this on the reference issue already. Saw this was open so I'm pasting here

Did some testing, on the Windows terminal, the characters show up correctly, but decode error. The issue is also on powershell, no characters and decode problem. On a debian based distro, it works, but it looks like it have some struggle to find the meta but that my be network problems by my side

**Comment by deepjyoti30** [comment]
@loulou310 So the charcaters are showing up properly for you but the error is from `simber` right?

Seems like an Windows issue to me.

**Comment by xotakfr** [comment]
Here's the error. It looks like it's from simber
```
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Python310\Scripts\ytmdl.exe\__main__.py", line 7, in <module>
  File "C:\Python310\lib\site-packages\ytmdl\main.py", line 582, in entry
    extract_data()
  File "C:\Python310\lib\site-packages\ytmdl\main.py", line 577, in extract_data
    main(args)
  File "C:\Python310\lib\site-packages\ytmdl\main.py", line 282, in main
    path = download(link, yt_title, args)
  File "C:\Python310\lib\site-packages\ytmdl\core.py", line 109, in download
    logger.info('Downloading {}{}{} in {}{}kbps{}'.format(
  File "C:\Python310\lib\site-packages\simber\logger.py", line 366, in info
    self._write(message, args, LEVEL_NUMBER)
  File "C:\Python310\lib\site-packages\simber\logger.py", line 203, in _write
    stream.write(
  File "C:\Python310\lib\site-packages\simber\stream.py", line 138, in write
    print(_formatted_out, **params)
  File "C:\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 48-54: character maps to <undefined>
```

**Comment by Niakr1s** [comment]
I fixed this error, by adding encoding to utf-8 to Logger._init_default_streams function:
```
self._streams.add(
            OutputStream(
                open(self._log_file, "a", encoding="utf-8"),
                ............
            ))
```
For example, I could download japanese music via ytmdl project, that caused such error before.

**Comment by deepjyoti30** [comment]
@Niakr1s Did you test this fix with Windows?

**Comment by Niakr1s** [comment]
> @Niakr1s Did you test this fix with Windows?

I didn't run tests, coz didn't have much time to dive deeper, but yes, on windows, with ytmdl, i could download j-pop song without errors, that caused "UnicodeEncodeError: 'charmap' codec bla bla bla" error before!

**Comment by deepjyoti30** [comment]
@Niakr1s Thanks for the confirmation! Just wanted to know if this fix works on WIndows since I don't have a Windows machine to test it right now.

**Comment by Niakr1s** [comment]
You are welcome :)

**Comment by deepjyoti30** [comment]
@Niakr1s This should be fixed in e1030a4
